pkgs/applications/misc/ganttproject-bin/default.nix

```diff
@@ -28,18 +30,20 @@ stdenv.mkDerivation rec {
       categories = "Office;Application;";
     };
 
+    javaOptions = []
+      ++ lib.optional enableAntiAliasing "-Dawt.useSystemAAFontSettings=on";
```

Comment:
Ok. Just read about your remark. What is a potential downside of anti-aliasing?

Reply:
Hey @Mic92, I know some users that are still on some Thinkpads, wasn't too sure if they would prefer the anti-aliasing thing by default. In my case it all looks pleasant on my 4K display, but when I had a lower-res machine, I often preferred keeping AA disabled because it just fuzzied up the screen. But yeah... I could just keep AA on as a default.

Reply:
@Mic92 and @Lassulus, I just removed the java options that I previously pushed as

```diff
diff --git a/pkgs/applications/misc/ganttproject-bin/default.nix b/pkgs/applications/misc/ganttproject-bin/default.nix
index 08b9f714588..0c91b7a055a 100644
--- a/pkgs/applications/misc/ganttproject-bin/default.nix
+++ b/pkgs/applications/misc/ganttproject-bin/default.nix
@@ -1,5 +1,7 @@
-{ stdenv, fetchzip, makeDesktopItem, makeWrapper
-, jre }:
+{ stdenv, lib, fetchzip, makeDesktopItem, makeWrapper
+, jre
+, enableAntiAliasing ? false
+}:
 
 stdenv.mkDerivation rec {
   name = "ganttproject-bin-${version}";
@@ -28,13 +30,18 @@ stdenv.mkDerivation rec {
       categories = "Office;Application;";
     };
 
+    javaOptions = []
+      ++ lib.optional enableAntiAliasing "-Dawt.useSystemAAFontSettings=on";
+
   in ''
     mkdir -pv "$out/share/ganttproject"
     cp -rv *  "$out/share/ganttproject"
 
     mkdir -pv "$out/bin"
     wrapProgram "$out/share/ganttproject/ganttproject" \
-      --set JAVA_HOME "${jre}"
+      --set JAVA_HOME "${jre}" \
+      --set _JAVA_OPTIONS "${builtins.toString javaOptions}"
+
     mv -v "$out/share/ganttproject/ganttproject" "$out/bin"
 
     cp -rv "${desktopItem}/share/applications" "$out/share"
```

such that users can handle this themselves if they need to.